src/components/RightSidebar.js

```diff
@@ -44,20 +44,19 @@ const RightSidebar = React.createClass({
   watchExpressionHeaderButtons() {
     const { expressionInputVisibility } = this.state;
     return [
-      debugBtn(
-        evt => {
-          evt.stopPropagation();
-          this.props.evaluateExpressions();
-        }, "domain",
-        "accordion-button", "Refresh"),
-      debugBtn(
-        evt => {
+      dom.button({
+        onClick: evt => {
           evt.stopPropagation();
           this.setState({
             expressionInputVisibility: !expressionInputVisibility
           });
-        }, "file",
-        "accordion-button", "Add Watch Expression")
+        }, className: "add-button accordion-button" }, "+"),
+      debugBtn(
+        evt => {
+          evt.stopPropagation();
+          this.props.evaluateExpressions();
+        }, "refresh",
+        "accordion-button", "Refresh")
```

Comment:
perhaps we don't need this accordion-class, and instead could use a scoped selector like `.expressions .hader .refresh`